src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceAccounts.java

```diff
@@ -0,0 +1,182 @@
+    }
+
+    /**
+     * Method to know if an account is saved.
+     * @return the new account name displayed in label.
+     */
+    public String newAccountSavedName() {
+        return this.webDriver.findElement(this.newAccountLabel).getText();
+    }
+
+    /**
+     * Method to set view in accounts list.
+     * @param accountView view list.
+     */
+    public void setAccountsViewList(final String accountView) {
+        Select select = new Select(this.webDriver.findElement(this.accountsViewList));
+        select.selectByValue(accountView);
+    }
+
+    /**
+     * Method to choose accounts view, if it is all accounts then press button go.
+     * @param accountView view.
+     * @param button to click.
+     */
+    public void chooseAccounts(final String accountView, final WebElement button) {
+        final int waitSeconds = 90;
+        WebDriverWait wait = new WebDriverWait(this.webDriver, waitSeconds);
+        WebElement searchOnDashboard = wait.until(ExpectedConditions.visibilityOfElementLocated(this.accountsViewList));
+        Select select = new Select(searchOnDashboard);
+        if ("All Accounts".equalsIgnoreCase(select.getFirstSelectedOption().getText().toString())) {
+            clickGoButton(button);
+        } else {
+            setAccountsViewList(accountView);
+        }
+    }
+
+    /**
+     * Method to get go button.
+     * @return button.
+     */
+    public WebElement getGoButton() {
+        return this.webDriver.findElement(this.goButton);
+    }
+
+    /**
+     * Method to click go button.
+     * @param button to click.
+     */
+    public void clickGoButton(final WebElement button) {
+        button.click();
+    }
+
+    /**
+     * Method to verify all accounts was selected.
+     * @param accountView list.
+     * @return list value.
+     */
+    public String isAllAccountsSelected(final String accountView) {
+        final int waitSeconds = 90;
+        WebDriverWait wait = new WebDriverWait(this.webDriver, waitSeconds);
+        WebElement searchOnDashboard = wait.until(ExpectedConditions.visibilityOfElementLocated(By.className("title")));
+        Select select = new Select(searchOnDashboard);
+        return select.getFirstSelectedOption().getText().toString();
+    }
+
+    /**
+     * Method to click edit link.
+     */
+    public void editAccount() {
+        WebElement editCustTable = webDriver.findElement(By.className("x-grid3-scroller"));
+        List<WebElement> tableRows = editCustTable.findElements(By.className("x-grid3-row-table"));
+        for (Iterator iterator = tableRows.iterator(); iterator.hasNext();) {
+            System.out.println("4");
+            WebElement row = (WebElement) iterator.next();
+            String x = ".x-grid3-row-table > tbody > tr > td >div[id='001f400000MSHJ7_ACTION_COLUMN']";
+            System.out.println("elements:" + row.findElement(By.cssSelector(x)).getText());
+            /*
+            if(row.getAttribute("id").contains("1010294")){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)